src/Parser/MistweaverMonk/Constants.js

```diff
@@ -0,0 +1,43 @@
+// Mistweaver Monk constants
+// Spells
+export const EFFUSE_HEAL_ID = 116694;
```

Comment:
Putting any kind of spell IDs here is no longer the way to go. Please add them to common/SPELL_ABILITIES